### PR TITLE
JIRA: ABC-290 Strip alembic behaviour instances when building unsupported targets.

### PR DESCRIPTION
**Pull request by tlaedre**
Topic for discussion: Strip scene alembic data to avoid serialization issues from code missing at runtime (mismatching type sizes).

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.formats.alembic?pullRequest=493) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.formats.alembic?pullRequest=493) before we can accept your contribution.<br/><hr/>**noreply@unity3d.com** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.formats.alembic?pullRequest=493) it.</sub>

**Review by clusty — APPROVED** [review]
Works for me.

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested on:
- 2021.2.7f1 (Windows host, build/run UWP)

File: com.unity.formats.alembic/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [2.3.0-pre.4] - 2022-01-02
 ### Added
 ### Changed
+- When building unsupported standalone targets, the Alembic Components get stripped from the build.
```
Comment (sebastienduverne) [inline com.unity.formats.alembic/CHANGELOG.md:10]: Suggestion:
`When building unsupported standalone targets, Unity excludes the Alembic components from the build.`

**Review by sebastienduverne — APPROVED** [review]
Changelog ok

**Pull request merged**